scripts/packages/postupgrade.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Just trying to understand why we are doing things differently here when the OS is alpine?
Is there a way to keep the old version of the agent running and let the user restart the agent afterwards when they want to switch versions?

Reply:
@dhurley alpine is following exactly the same pattern here as RHEL/Debian; it just has slightly different schema of package scripts, hence some actions were moved to another specific stages. The way agent is using to build linux packages is quite confusing and obfuscating (yes, I mean NFPM) - I strongly recommend to consider moving pkg building to "natural" tooling (rpmbuild/debuild/abuild, etc), that will resolve many ambiguities that are currently in place with NFPM.